public/js/lean.js

```diff
@@ -1,41 +1,96 @@
 apos.utils.widgetPlayers['apostrophe-forms-google-address-field'] = function(el, widget, options) {
-  var scriptSrc = 'https://maps.googleapis.com/maps/api/js?key=' + widget.googleApiKey + '&libraries=places';
-  var alreadyLoaded = document.querySelector('script[src="' + scriptSrc + '"]');
-  if (!alreadyLoaded && widget.googleApiKey) {
-    var googleScript = document.createElement('script');
-    googleScript.setAttribute('src', scriptSrc);
-    document.head.appendChild(googleScript);
+  var formsWidget = apos.utils.closest(el, '[data-apos-widget="apostrophe-forms"]');
+  if (!formsWidget) {
+    // Editing the form in the piece modal, it is not active for submissions
+    return;
   }
 
-  var input = el.querySelector('input[name=' + widget.fieldName + ']');
-  input.addEventListener('input', function () {
-    var google = window.google;
+  var googleApiKey = el.querySelector('[data-apos-forms-google-api-key]').dataset.aposFormsGoogleApiKey;
+  if (googleApiKey) {
```

Comment:
For readability I would suggest returning out early if the key is missing rather than going through all the work then having an `else` case with only an error message. That's optional.

Reply:
Done